test/index.spec.js

```diff
@@ -51,3 +96,29 @@ test('named export | throw because wrong value type (undefined)', () => {
 
   expect(result).toThrow('value has to be typeof: \'string\' but got typeof: \'undefined\'');
```

Comment:
We should also update the value of this test as well.

Reply:
done.